ksqldb-engine/src/main/java/io/confluent/ksql/util/BinPackedPersistentQueryMetadataImpl.java

```diff
@@ -382,7 +382,7 @@ public void onStateChange(final State newState, final State oldState) {
 
   @Override
   public void close() {
-    sharedKafkaStreamsRuntime.stop(queryId, true);
```

Comment:
Missed this in my previous PR. https://github.com/confluentinc/ksql/pull/8755

Reply:
Catching up here - how do you know for sure that this isn't a `createOrReplace`?

Reply:
A create of replace uses the `stop(boolean)`, you can see that [here](https://github.com/confluentinc/ksql/blob/b8df77716226efcc93aa932e0ce8c0f58381cb0e/ksqldb-engine/src/main/java/io/confluent/ksql/query/QueryRegistryImpl.java#L446)